destiny/main/etl/extract.py

```diff
@@ -58,93 +71,333 @@ def extract_summoners(p_session, nb_sum_needed):
             tier = league['tier']
             for entrie in league['entries']:
                 if entrie['playerOrTeamId'] not in summoners_stack and len(summoners_stack) < nb_sum_needed:
-                    summoners_stack.add(entrie['playerOrTeamId'])
-                    summoner_id = int(entrie['playerOrTeamId'])
+                    summoners_stack.add(entrie.get('playerOrTeamId'))
```

Comment:
This is not pythonic and I don't think you should use the `get` method of `dict` type. This should be used if you need a default value or if you are not bothered by `None` type objects in your set, which is not the case here, I think. If you want to prevent `KeyError` exception to occur, you should use a `try... except` block.
https://stackoverflow.com/questions/11041405/why-dict-getkey-instead-of-dictkey